### ISSUE TITLE
Reboot of Homatics Box R 4K Plus Stops PL From being Default launcher 

### ISSUE THREAD

**Issue by xxxsskxxx** [spocky/miproja1#112]
Hi

Thank you for the app!

I have 4.54 installed on my Homatics Box R 4K Plus.

I reboot the device,and then the launcher reverts back to the stock launcher, and accessibility settings are lost.

Even when I re-enable accessibility settings, and toggle 'overide control for home' checkbox, and press home, it takes me back to the stock launcher.

Crash reporting is enabled.

Let me know if you need anymore details

**Comment by spocky** [comment]
Either this box disabled the accessibility on purpose upon each boot (Amazon devices are known for actively fighting against device modifications they don't approve, I've not heard this about this device, but it can't exclude it), or there's a crash somewhere that kills the accessibility. My crash logs don't show any crash regarding the accessibility service (which didn't mean there aren't).

When you enable the accessibility service again, obese make sure (in Projectivy's accessibility menu) that it's been really reenabled. There is a strange behavior in Android that sometimes requires you to disable and reenable the accessibility service mentally before it is really running again.

**Comment by xxxsskxxx** [comment]
Yeah it's strange. The only way to get it back to stick is if I uninstall and reinstall the application. 

It's absolutely fine and accessibility takes upon first setup

When I reboot, and then re enabled it, it looks like it's enabled and taken 

But it hasn't as when I re check the settings the launcher reports back 'No'

I'll try and do a video with what's happening 

Cheers

**Comment by xxxsskxxx** [comment]
This has not happened in quite sometime, and will deem it a FW issue with the device, Android TV 14 is due on the box from 9th December, will keep you posted on how that performs :)

**Comment by spocky** [comment]
Thanks for getting back and closing the issue
